vendor/k8s.io/gengo/examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -184,10 +184,18 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 
 		if pkgNeedsGeneration {
 			glog.V(3).Infof("Package %q needs generation", i)
+			path := pkg.Path
```

Comment:
Can you give an example before and after for this chunk when you open the upstream.  It's not obvious at a glance what this is doing and I'm wondering if you already added similar code elsewhere.